src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -63,6 +63,8 @@ public static String getEditPersonDescriptorDetails(EditPersonDescriptor descrip
         }
         descriptor.getUpcoming()
                 .ifPresent(upcoming -> sb.append(PREFIX_UPCOMING).append(upcoming.toString()).append(" "));
+        descriptor.getLastcontact().ifPresent(lastContact -> sb.append(PREFIX_LASTCONTACT).append(" ")
+                .append(lastContact.getDateTimeString()));
```

Comment:
Perhaps this part can follow Yun Chao's format? 🤔

Reply:
> Perhaps this part can follow Yun Chao's format? 🤔
Sure bro